.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
 
   # Lint: Dockerfile
   - repo: https://github.com/hadolint/hadolint.git
-    rev: v2.6.1
```

Comment:
Hadolint 2.6.1 is also not yet available.
https://hub.docker.com/r/hadolint/hadolint/tags?page=1&ordering=last_updated